### PR TITLE
Fix most of the broken links

### PR DESCRIPTION
**Pull request by mdamien**
Methodology was to use this script: https://gist.github.com/mdamien/7b71ef06f49de1189fb75f8fed91ae82

And then find the proper link by hand (I should automate this part too)

There are a few links in `interviews.html` that I can't find a proper link for:

  - http://www.xs4all.nl/~johannab/DP-GvR.html
  - http://zweistein.host.sk/hosting/nplejic/scitech/gvr_interview.html
  - http://gnu-friends.org/story/2002/2/25/17162/1443

Have a good day :tulip:

### CODE REVIEW

File: Resume.html [hunk 1]
```diff
@@ -63,27 +63,27 @@
 elaborate example, I wrote <a href="http://grail.sf.net">Grail</a>,
 the first web browser written in Python.  During this time I also
 wrote a funding proposal, <a
-href="http://www.python.org/cp4e/">Computer Programming for
+href="https://web.archive.org/web/20030217160408/http://www.python.org:80/cp4e/">Computer Programming for
```
Comment (gvanrossum) [inline Resume.html]: This is actually at https://www.python.org/doc/essays/cp4e/

File: Resume.html [hunk 2]
```diff
@@ -63,27 +63,27 @@
...
 Everybody</a>, that was funded by DARPA.
 
 <P>From mid October till mid December 1994 I was a guest researcher at
 <A HREF="http://www.nist.gov">NIST</A>, working on <A
 HREF="http://www.python.org">Python</A>.  NIST sponsored my visit to
 the Usenix Symposium on Very High Level Languages in Santa Fe and
-organized the <A HREF="http://www.python.org/workshops/">First Python
+organized the <A HREF="https://web.archive.org/web/20130201175839/http://www.python.org/workshops/">First Python
```
Comment (gvanrossum) [inline Resume.html]: http://legacy.python.org/workshops/

File: Resume.html [hunk 3]
```diff
@@ -98,18 +98,18 @@
 <LI>From 1986 till 1991 I was with the Amoeba project, headed by <A
 HREF="http://www.pegasus.esprit.ec.org/~sape/">Sape
```
Comment (gvanrossum) [inline Resume.html]: Your script missed this one.

File: Resume.html [hunk 4]
```diff
@@ -63,27 +63,27 @@
...
+organized the <A HREF="https://web.archive.org/web/20130201175839/http://www.python.org/workshops/">First Python
 Workshop</A>.
 
 <H3>Previous Work outside the US</H3>
 
 <P>From 1991 till 1995 I worked in the multimedia group at <A
 HREF="http://www.cwi.nl/">CWI</A>, headed by <A
-HREF="http://www.cwi.nl/cwi/people/Dick.Bulterman.html">Dick
+HREF="http://homepages.cwi.nl/~dcab/">Dick
 Bulterman</A>.  The group is (still) working on authoring software for
```
Comment (gvanrossum) [inline Resume.html]: s/is (still)/was/

File: Resume.html [hunk 5]
```diff
@@ -63,27 +63,27 @@
...
 hypermedia presentations (both implementations and theoretical models)
 and on operating system and network support for multimedia and
 hypermedia, in particular synchronization of independent streams.
 They maintain a <A
```
Comment (gvanrossum) [inline Resume.html]: past tense

File: Resume.html [hunk 6]
```diff
@@ -63,27 +63,27 @@
...
-HREF="http://www.cwi.nl/ftp/mmpapers/">directory</A> containing
+HREF="https://web.archive.org/web/19990225081903/http://www.cwi.nl:80/ftp/mmpapers/">directory</A> containing
 compressed Postscript of publications by the group.  Most of the
 group's implementation work (even after my departure) is done in <A
```
Comment (gvanrossum) [inline Resume.html]: is -> was

File: Resume.html [hunk 7]
```diff
@@ -98,18 +98,18 @@
...
+HREF="https://web.archive.org/web/20061006040156/http://www.cs.vu.nl:80/vakgroepen/cs/top.html">Computer Systems
 Group</A> of the <A
-HREF="http://www.cs.vu.nl/vakgroepen/informatica.html">Department of
+HREF="https://web.archive.org/web/20050212074025/http://www.cs.vu.nl/vakgroepen/informatica.html">Department of
 Computer Science</A> of the <A HREF="http://www.vu.nl/">Free
 University of Amsterdam</A>.<P>
 
 <LI>From 1982 till 1986 I was a member of the ABC group, headed by <A
-HREF="http://www.cwi.nl/cwi/people/Lambert.Meertens.html">Lambert
+HREF="http://www.kestrel.edu/home/people/meertens/">Lambert
```
Comment (gvanrossum) [inline Resume.html]: That link doesn't work.

File: Resume.html [hunk 8]
```diff
@@ -130,14 +130,14 @@
 <h3>Awards</h3>
 
 <p>In June 2013 Python won the highly competitive
-<a href="http://www.nederlandict.nl/index.shtml?id=12854&ch=ICT">Dutch
+<a href="https://web.archive.org/web/20141105123845/http://www.nederlandict.nl/index.shtml?id=12854&ch=ICT">Dutch
```
Comment (gvanrossum) [inline Resume.html]: Use http://www.commit-nl.nl/news/python-winnaar-commit-award-2013

File: Resume.html [hunk 9]
```diff
@@ -130,14 +130,14 @@
...
 COMMIT/ Award</a>.
 
 <p>In July 2007 I was awarded the USENIX
 <a href="http://www.usenix.org/about/stug.html">STUG Award</a>.
 
 <p>In October 2006 I was elected
-<a href="http://awards.acm.org/homepage.cfm?awd=157">ACM
+<a href="https://web.archive.org/web/20061205221308/http://awards.acm.org:80/homepage.cfm?awd=157">ACM
```
Comment (gvanrossum) [inline Resume.html]: https://awards.acm.org/distinguished-members/award-winners?year=2006&award=157&region=North+America&submit=Submit&isSpecialCategory=

File: Resume.html [hunk 10]
```diff
@@ -150,7 +150,7 @@
 Unix and Open Systems.
 
 <P>In February 2002 I received the <a
-href="http://www.fsf.org/press/2002-02-16-FSF-Award.html">Free
+href="https://web.archive.org/web/20020220040233/http://www.fsf.org:80/press/2002-02-16-FSF-Award.html">Free
```
Comment (gvanrossum) [inline Resume.html]: https://www.gnu.org/award/2001/2001.html

File: Resume.html (hunk 7 above)
Comment (mdamien) [inline Resume.html]: It works for me, not sure why it wouldn't work for you

**Comment by mdamien** [comment]
I updated with your links, otherwise I tend to prefer archive.org for things like awards or interviews since it's not gonna change

File: Resume.html (hunk 7 above)
Comment (TiberiumN) [inline Resume.html]: Doesn't work for me too

Comment (Dorian) [inline Resume.html]: screen shot 2017-07-05 at 3 11 45 pm

Comment (mdamien) [inline Resume.html]: ok, updated the link to the Utrecht University profile instead

**Comment by gvanrossum** [comment]
Thanks! It's sad how many links move around. That was not the original intention of the designers of the web.

**Pull request merged**